modules/nf-core/mtmalign/align/main.nf

```diff
@@ -0,0 +1,46 @@
+
+
+process MTMALIGN_ALIGN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::mtm-align=20220104"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mtm-align:20220104--h4ac6f70_0':
+        'biocontainers/mtm-align:20220104--h4ac6f70_0' }"
+
+    input:
+    tuple val(meta), path('*.pdb', arity: '2..*')
+
+    output:
+    tuple val(meta), path("./mTM_result/result.fasta")  , emit: alignment
+    tuple val(meta), path("./mTM_result/result.pdb")    , emit: structure
```

Comment:
```suggestion
    tuple val(meta), path("./mTM_result/*.pdb")    , emit: structure
```

Reply:
(mtmalign copies the input structures into the results directory)

Reply:
Maybe we can then move that file into ${prefix}.pdb ? 

I think it's important to have the possibility ti have the output files named correctly